template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Program.cs

```diff
@@ -14,14 +16,31 @@ public static void Main(string[] args)
             var host = new WebHostBuilder()
                 .UseKestrel()
                 .UseContentRoot(Directory.GetCurrentDirectory())
-                .UseIISIntegration()
-                .UseStartup<Startup>()
-#if (IncludeApplicationInsights)
-                .UseApplicationInsights()
+                .ConfigureConfiguration((context, configBuilder) => {
+#if (IndividualAuth || OrganizationalAuth)
+                    configBuilder.AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
```

Comment:
Should the `AddJsonFile` on this line move down to the next line, and be indented? Then the next line's `AddJsonFile` should also be indented.